.travis.yml

```diff
@@ -1,9 +1,7 @@
 dist: trusty
 language: node_js
 node_js:
-  - '6'
-  - '5'
-  - '4'
+  - '7'
```

Comment:
I'd like restypie to remain compatible until v4. We're not the only ones using it.